examples/1_hypercharge_postgres/dozer-config.yaml

```diff
@@ -41,14 +61,13 @@ sources:
 endpoints:
   - name: stocks
     path: /stocks
-    table_name: stocks
```

Comment:
why dont you let one API connect directly to source but select the columns at the source level and another transformed using SQL. 

And put comments near the endpoints explaining how they work ?

Reply:
sure - will do